admin_guide/overcommit.adoc

```diff
@@ -330,35 +330,34 @@ required.
 [[disabling-swap-memory]]
 === Disabling Swap Memory
 
-It is important to understand that oversubscribing the physical resources on a
-node affects resource guarantees the Kubernetes scheduler makes during pod
-placement. For example, suppose two guaranteed pods have reached their memory
-limit. Each container may start using swap. Eventually, if there is not enough
-swap space, processes in the pods can be terminated (due to the system being
-oversubscribed).
+You can disable swap by default on your nodes in order to preserve quality of
+service guarantees. Otherwise, physical resources on a node can oversubscribe,
+affecting the resource guarantees the Kubernetes scheduler makes during pod
+placement. 
```

Comment:
Sounds good. The initial BZ is about this contradiction, so I'll check if that's enough for Eric ( @TheDiemer ) and continue with the rest of the comments. Thanks, all.